joomla/joomla-cms
# [5.1] Fix user field validation on change

### Summary of Changes

This PR fixes error validation on a "User" field when a user is selected.

### Testing Instructions

1. Got to `administrator/index.php?option=com_messages&view=messages`
2. Click "New"
3. Focus on the "Recipient" user input field
4. Focus away from the "Recipient" user input field (you'll see a red error message)
5. Select a user

### Actual result BEFORE applying this Pull Request

Field remained red with the error message

### Expected result AFTER applying this Pull Request

Error message is removed and field border turns green
